src/state/stake/hooks.ts

```diff
@@ -1692,49 +1444,24 @@ export function useOldStakingInfo(
           !earnedAmountState?.loading &&
           // always need these
           totalSupplyState &&
-          !totalSupplyState.loading &&
-          periodFinishState &&
-          !periodFinishState.loading
+          !totalSupplyState.loading
         ) {
-          if (
-            balanceState?.error ||
-            earnedAmountState?.error ||
```

Comment:
What replaces all of this code that was removed?

Reply:
It was returning many console errors because of this.